docs/8.abstraction/chain-signatures.md

```diff
@@ -7,104 +7,160 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {CodeTabs, Language, Github} from "@site/src/components/codetabs"
 
-Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of cross-chain accounts, data, and assets to one NEAR account.
+NEAR accounts can safely control accounts across different blockchain protocols, allowing each NEAR account to have true ownership of cross-chain data and assets.
 
-In this document we cover the steps necessary to sign a transaction for another chain, we recommend you to read our [**overview of Chain Signatures**](../1.concepts/abstraction/chain-signatures.md) before, as well as some of its [**use cases**](../1.concepts/abstraction/signatures/use-case.md).
+![chain-signatures](/docs/assets/welcome-pages/chain-signatures-overview.png)
+_Diagram of a chain signature in NEAR_
 
-:::info
+There are five steps involved on Chain Signatures:
 
-See our [web-app example](https://github.com/near-examples/near-multichain) and [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx) showing how a NEAR account can create an Ethereum testnet transaction.
+1. [Deriving the Foreign Address](#1-deriving-foreigner-the-address) - Reconstruct the address that will be controlled on the target blockchain
+2. [Creating a Payload](#2-creating-the-payload) - Create the transaction / message that we want to sign for the target blockchain
+3. [Requesting a Signature](#3-requesting-the-signature) - Call the NEAR multichain contract, requesting to sign the payload
+4. [Reconstructing the Signature](#4-reconstructing-the-signature) - Reconstruct the signature from the MPC service's response
+5. [Relaying the Signed Payload](#5-relaying-the-signature) - The signed payload is then sent to the destination chain for execution.
 
+:::info Looking for examples?
+See our [web-app example](https://github.com/near-examples/near-multichain) and [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx) showing how a NEAR account can create transactions on Ethereum.
```

Comment:
@think-in-universe posted that there's a bug on the BOS component, and he made a fork with the fix:
https://test.near.social/bot.testnet/widget/chainsig-sign-eth-tx

[Reference](https://pagodaplatform.slack.com/archives/C03PKHB82QP/p1709914663561569?thread_ts=1709603754.834769&cid=C03PKHB82QP)

Reply:
Agree, lets use it